amplify.yml

```diff
@@ -14,7 +14,7 @@ frontend:
           else
             yarn run build:staging;
           fi
-  artifacts:
+  Nerdpack items:
```

Comment:
Yep, let's focus only on the `.mdx` files which contain docs/guide content. These js files are out of scope for content changes.